python-checks/src/test/resources/checks/nonCallableCalledNoIssues.py

```diff
@@ -1,3 +1,6 @@
 
 def param_with_type_hint(a: int):
-    a() # OK
\ No newline at end of file
+    a() # OK
+
+def call_unknown_value():
+    (~s)(X) # Ok
```

Comment:
_minor_ I find it not explicit what you want to test here. What matters is not really the fact that `s` is unknown but that there is a unary ~ operator applied on `s` right? 
I suggest adding a test in `nonCallableCalled` called `def callable_with_unary_operator` and checking that not issue are raised when `+,-,~...` are applied

Reply:
Now that I read the all PR I understand why you called the test function like this. Still, I think it will be clearer for someone who hasn't the context to rewrite this test to be more explicit